cli/Valet/Brew.php

```diff
@@ -7,6 +7,8 @@
 
 class Brew
 {
+    const SUPPORTED_PHP_VERSIONS = ['php', 'php@7.2', 'php@7.1', 'php@7.0', 'php@5.6'];
```

Comment:
Like mentioned in my comment down below it's important we maintain support for the old naming convention for people who haven't upgraded Homebrew (like me 😜)

Reply:
@adamwathan Add the names for the old versions